modules/update-upgrading-cli.adoc

```diff
@@ -75,19 +61,21 @@ VERSION IMAGE
 4.10.12 quay.io/openshift-release-dev/ocp-release@sha256:f77f4f75c1e1a4ddd0a0355f298a834db3473fd9ca473235013e9419d1df16db
 4.10.13 quay.io/openshift-release-dev/ocp-release@sha256:4f516616baed3cf84585e753359f7ef2153ae139c2e80e0191902fbd073c4143
 ----
+[NOTE]
+*For details and information on how to perform an `EUS-to-EUS` channel upgrade, please refer to the link:https://docs.openshift.com/container-platform/4.12/updating/preparing-eus-eus-upgrade.html[Preparing to perform an EUS-to-EUS upgrade] page.
 
-. Based on your organization requirements, set the upgrade channel to `stable-{product-version}`, `fast-{product-version}`, or `eus-{product-version}`:
+. Based on your organization requirements, set the upgrade channel to `stable-4.9`, `fast-4.9`, or `eus-*`:
```

Comment:
We should say `Based on your organization requirements, set the appropriate upgrade channel. For example: `stable-4.9`, `fast-4.9`, or `eus-*`. Refer to the channel documentation (link to the channel docs[1]) for more information about channels. 
[1] https://docs.openshift.com/container-platform/4.12/updating/understanding-upgrade-channels-release.html